### ISSUE TITLE
Jupyter protip

### ISSUE THREAD

**Issue by GreatEmerald** [GeoScripting-WUR/PythonProgramming#1]
It would be nice to mention somewhere that since Jupyter is a web server, it can be accessed directly through SSH as
```bash
ssh -N -L8888:localhost:8888 username@VDIname
```
And then going to http://localhost:8888. That should make students' lives a bit nicer.

Hm, it would overall be nice to have something like "protip" boxes of things that are useful but students are not required to know them...

**Comment by Davidswinkels** [comment]
It would be good to add this knowledge, but are you sure this works on the VDIs?
It does not directly work on my local Ubuntu 18.04 machine.
Error: ssh: connect to host david-n56vm port 22: Connection refused

If VDIs are available again, can you (@GreatEmerald) test it and then also add it as a protip?
It would be very nice to have an explanation of SSH.

**Comment by GreatEmerald** [comment]
Yes, it does work on VDIs, SSH is enabled by default. You just need to know your VDI name or IP.
